### ISSUE TITLE
[Bug] importing covisirphy is slow because importing sklearn when "import covsirphy"

### ISSUE THREAD

**Issue by lisphilar** [lisphilar/covid19-sir#1265]
### Checkbox

- [X] I agree to follow [code of conduct](https://lisphilar.github.io/covid19-sir/CODE_OF_CONDUCT.html) and to engage in discussion actively.
- [X] I agree to create a new issue with the other templates if this is not a bug.
- [X] I used [the latest version of covsirphy](https://lisphilar.github.io/covid19-sir/markdown/INSTALLATION.html).

### Summary

`covsirphy.Evaluator` uses `sklearn.metrics` and importing this takes about 5 seconds. `sklearn` is used only in `Evaluator` and rewriting metrics codes with `numpy` may reduces the importing time.

### Reproducible example script

```python
time python -c "import covsirphy"
time python -c "import sklearn"
time python -c "import numpy"
```

### The current outputs

```python
time python -c "import covsirphy"

real    0m18.736s
user    0m3.424s
sys     0m3.629s

$ time python -c "import sklearn"

real    0m5.610s
user    0m1.097s
sys     0m1.856s

$ time python -c "import numpy"

real    0m0.854s
user    0m0.214s
sys     0m0.643s
```

### Expected outputs

```python
`time python -c "import covsirphy"` takes 13 seconds (=18 - 5) at in my environment.
```

### Environment

```markdown
- CovsirPhy version: 3.0.0-dev
- Python version: 3.10.7
```

### Package manager (required if installation issue)

poetry

### Platform (required if installation issue)

Ubuntu

**Comment by lisphilar** [comment]
ME: maximum residual error
MAE: https://pystyle.info/ml-regression-criteria/#outline__2_5
MSE: https://www.askpython.com/python/examples/rmse-root-mean-square-error 
MSLE: https://pystyle.info/ml-regression-criteria/#outline__2_3
MAPE: https://www.askpython.com/python/examples/mape-mean-absolute-percentage-error
RMSE: https://www.askpython.com/python/examples/rmse-root-mean-square-error
RMSLE: https://gist.github.com/Tafkas/7642141
R2: https://www.adamsmith.haus/python/answers/how-to-calculate-r-squared-with-numpy-in-python

**Comment by lisphilar** [comment]
```Python
import numpy as np
    # Metrics: {name: (function(x1, x2), whether smaller is better or not)}
    _METRICS_DICT = {
        "ME": (lambda x1, x2: np.max(np.abs(x2 - x1)), True),
        "MAE": (lambda x1, x2: np.mean(np.abs(x2 - x1)), True),
        "MSE": (lambda x1, x2: np.mean(np.square(x2 - x1)), True),
        "MSLE": (lambda x1, x2: np.mean(np.square(np.log1p(x2) - np.log1p(x1))), True),
        "MAPE": (lambda x1, x2: np.mean(np.abs((x2 - x1) / x1)) * 100, True),
        "RMSE": (lambda x1, x2: np.sqrt(np.mean(np.square(x2 - x1))), True),
        "RMSLE": (lambda x1, x2: np.sqrt(np.mean(np.square(np.log(x2 + 1) - np.log(x1 + 1)))), True),
        "R2": (lambda x1, x2: np.corrcoef(x1, x2)[0, 1]**2, False),
    }
```

**Comment by lisphilar** [comment]
In-effective on importing time because `sklearn` is used in anather class for PCA.

**Comment by lisphilar** [comment]
Currently, `make importtime` command (#1290) shows
![image](https://user-images.githubusercontent.com/7270139/197352694-6a6d49f7-a441-4344-8da7-a8268be62e0f.png)

**Comment by lisphilar** [comment]
With #1291,
- use `numpy` instead of `sklearn.metrics` in `Evaluator` class
- import `autots` and `pca` inside methods because they have `sklearn` as a dependency, making importing time slow

![image](https://user-images.githubusercontent.com/7270139/197353265-823bdccd-6084-4324-89dc-ec311273bd81.png)

**Comment by lisphilar** [comment]
In my environment at this time,
- before: 27 seconds in total
- after: 11 seconds in total
